articles/azure-fluid-relay/quickstarts/quickstart-dice-roll.md

```diff
@@ -68,6 +68,7 @@ const serviceConfig = {
 
 > [!WARNING]
 > During development, you can use `InsecureTokenProvider` to generate and sign authentication tokens that the Azure Fluid Relay service will accept. However, as the name implies, this is insecure and should not be used in production environments. The Azure Fluid Relay resource creation process provides you with a secret key which can be used to sign secure requests. **To ensure that this secret doesn't get exposed, this should be replaced with another implementation of ITokenProvider that fetches the token from a secure, developer-provided backend service prior to releasing to production.**
+> One secure approach is outlined in ["How to: Write a TokenProvider with an Azure Function"](../how-tos/azure-function-token-provider.md).
```

Comment:
For this line break to display correctly, you need to add a blank line between the paragraphs. Because this is in a warning, the "blank" line will have a `>` at the start, but will otherwise be blank.